[CI] testGetUsedMemoryWithZeroTotal & testGetUsedSwapWithZeroTotal fail

It reproduces, it seems we are passing negative values to the method:

```
org.elasticsearch.monitor.os.OsStatsTests > testGetUsedMemoryWithZeroTotal FAILED
    java.lang.AssertionError: expected free memory to be positive, got: 0
        at __randomizedtesting.SeedInfo.seed([B6A90649037E1E15:376E6629FDCFC798]:0)
        at org.elasticsearch.monitor.os.OsStats$Mem.<init>(OsStats.java:253)
        at org.elasticsearch.monitor.os.OsStatsTests.testGetUsedMemoryWithZeroTotal(OsStatsTests.java:87)
REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.monitor.os.OsStatsTests.testGetUsedMemoryWithZeroTotal" -Dtests.seed=B6A90649037E1E15 -Dtests.security.manager=true -Dtests.locale=be-BY -Dtests.timezone=America/Tortola -Druntime.java=11

org.elasticsearch.monitor.os.OsStatsTests > testGetUsedSwapWithZeroTotal FAILED
    java.lang.AssertionError: expected free swap to be positive, got: 0
        at __randomizedtesting.SeedInfo.seed([B6A90649037E1E15:B0847D9DA4444DA6]:0)
        at org.elasticsearch.monitor.os.OsStats$Swap.<init>(OsStats.java:192)
        at org.elasticsearch.monitor.os.OsStatsTests.testGetUsedSwapWithZeroTotal(OsStatsTests.java:92)
REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.monitor.os.OsStatsTests.testGetUsedSwapWithZeroTotal" -Dtests.seed=B6A90649037E1E15 -Dtests.security.manager=true -Dtests.locale=be-BY -Dtests.timezone=America/Tortola -Druntime.java=11
```

--- a/server/src/test/java/org/elasticsearch/monitor/os/OsStatsTests.java
+++ b/server/src/test/java/org/elasticsearch/monitor/os/OsStatsTests.java
@@ -84,12 +84,12 @@ public class OsStatsTests extends ESTestCase {
     }
 
     public void testGetUsedMemoryWithZeroTotal() {
-        OsStats.Mem mem = new OsStats.Mem(0, randomLong());
+        OsStats.Mem mem = new OsStats.Mem(0, randomNonNegativeLong());
         assertThat(mem.getUsed().getBytes(), equalTo(0L));
     }
 
     public void testGetUsedSwapWithZeroTotal() {
-        OsStats.Swap swap = new OsStats.Swap(0, randomLong());
+        OsStats.Swap swap = new OsStats.Swap(0, randomNonNegativeLong());
         assertThat(swap.getUsed().getBytes(), equalTo(0L));
     }
 }

